src/components/Clients/ClientListTableRow.jsx

```diff
@@ -43,6 +43,7 @@ const ClientListTableRow = ({ labelId, client, state, dispatch }) => {
   const handleSelectClient = async (clientToSelect) => {
     runNotification(`Client "${clientToSelect.person}" selected.`, 3, state, dispatch);
```

Comment:
does it always run line 44? Even if they are already selected?  

If so, it shouldn't. This should be an if/else no?

Reply:
Not really sure what kind of JS voodoo happening or the function calls are just really fast, it seems to run fine.

https://github.com/codeforpdx/PASS/assets/14917816/7992ad1c-c287-4377-9342-c09e786780e5

But you're right. Line 44 always seem to run regardless of condition. This would probably break if something is slowing down the function calls. I'm moving this to after the if statement.